app/services/security/__test__/cspHeader.test.ts

```diff
@@ -0,0 +1,71 @@
+import { getCspHeader } from "../cspHeader.server";
```

Comment:
For me the `it` import is missing within VSCode...
```suggestion
import { it } from "vitest";
import { getCspHeader } from "../cspHeader.server";
```